creator-node/src/utils/requestRange.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Gets range request headers off of a request object
+ * Drop in replacement for `req.range()` which does not
+ * support ranges without a specified `end`.
+ *
+ * This is a widely supported pattern, e.g.
+ * https://musicpartners.sonos.com/node/465
+ *
+ * @param {Request} req express request object
+ *
+ * @returns
+ *  { start, end } where start and end are integer byte values
+ *  or
+ *  null if the request could not be parsed
+ */
+const getRequestRange = (req) => {
+  const header = req.header('range')
+  if (!header) return null
+
+  // Format: bytes=start-end
+  // e.g. bytes=1024-2048
```

Comment:
from the sonos link it looks like Range is `Range:<start>-<end>` while `Content-Range: bytes <start>-/<size>`. Range doesn't have a prefix before it's start value. Is that intentional?

Reply:
MDN says it has bytes https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range

Reply:
I think that may be an issue on the sonos docs side. I think bytes is expected for Range
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Range

Also see https://github.com/tumtumtum/StreamingKit/blob/eb9268e83ea0c0cea6a1c7af16559de455d4326e/StreamingKit/StreamingKit/STKHTTPDataSource.m#L605

But! It should be in Content-Range too look like, so added than and added a unit tested helper.